### ISSUE TITLE
add vite-raw-plugin.js

### ISSUE THREAD

**Issue by black-sec** [project-nv/night-vision#95]
### Describe the bug

Hello, thank you for your good library
I have a question, how can I add vite-raw-plugin.js to my project so that I can create a new indicator, unfortunately, I am a beginner.

And doesn't this library have a toolbar tool?

### Reproduction

www

### Javascript Framework

no-framework (vanilla-js)

### Validations

- [X] Read the [docs](https://nightvision.dev/guide/intro/getting-started.html).
- [X] Check that there isn't [already an issue](https://github.com/project-nv/night-vision/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a NightVision issue and not a framework-specific issue.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by C451** [comment]
Updated the docs. Nope, no toolbar for now

**Comment by black-sec** [comment]
Thank you, I solved the problem, now I have a question
I wrote an indicator in TradingView, which is a combination of ema, RSI, divergenci, macdi, etc. indicators, which gives a buy or sell label on the chart when the conditions are correct.
Now I can implement that indicator by this library. Actually, I have been reading the library for a few days and so far I was able to implement the live data, but the part of making the indicator is very unclear to me, like the calculations I had in Pine Script or how to draw labels on the chart.
Thank you as much as possible
